docs/releases/4.0.txt

```diff
@@ -236,6 +236,11 @@ Management Commands
 
 * On PostgreSQL, :djadmin:`dbshell` now supports specifying a password file.
 
+* Internally the :djadmin:`shell` command now respects
+  :py:data:`sys.__interactivehook__` at startup. Primary this allows loading
+  shell history between interactive sessions. As a consequence, ``readline`` is
+  no longer loaded if running in *isolated* mode.
```

Comment:
I find cross reference links extremely helpful. Others find them unnecessary. Up to y'all.

```suggestion
  :ref:`shell history <python:tut-keybindings>` between interactive sessions. As a consequence, :py:mod:`readline` is
  no longer loaded if running in :ref:`isolated <python:init-isolated-conf>` (:option:`python:-I`) mode.
```

Reply:
This seems a bit overkill to me here. 

Whilst linking linking to `sys.__interactivehook__` is something that folks might learn something from here, I don't take it that we're teaching about these other items. Folks know what shell history and `readline` are, and we're not really advocating isolated mode. (Rather making sure you know we changed it.) 

But 🤷